### PR TITLE
feat: fix some bugs

### PR DESCRIPTION
**Pull request by angelia-yuqi-personal**
PORTAL-1615
PORTAL-1616
PORTAL-1644
PORTAL-1645
PORTAL-1661

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/294)

### CODE REVIEW

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1616 切换网络后，Address 发生改变，需要有一个 Toast 提示](https://app.clickup.com/t/3719212/PORTAL-1616)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1645 Transaction Submit 成功页咱不用要了](https://app.clickup.com/t/3719212/PORTAL-1645)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1644 Custom Nonce 我们给个建议的值吧，当前这笔交易的 Nonce](https://app.clickup.com/t/3719212/PORTAL-1644)

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1615 Send Transaction 没有输入值的时候不要提示错误](https://app.clickup.com/t/3719212/PORTAL-1615)

File: packages/estimate-tx/index.js [hunk 1]
```diff
@@ -151,50 +156,53 @@ export const cfxEstimate = async (
...
+
+  // wait for all those values
+  await Promise.all(promises)
+
   // simple send tx, gas is 21000, storageLimit is 0
   if (to && (!data || data === '0x')) {
+    const clcGasPrice = customGasPrice || gasPrice
+    const clcGasLimit = customGasLimit || '0x5208' /* 21000 */
+    const clcStorageLimit = customStorageLimit || '0x0'
     await Promise.all(promises)
```
Comment (yqrashawn) [inline packages/estimate-tx/index.js:179]: remove this line

**Review by yqrashawn — CHANGES_REQUESTED** [review]

File: packages/estimate-tx/index.js (hunk 1 above)
Comment (angelia-yuqi-personal) [reply packages/estimate-tx/index.js]: done

**Review by yqrashawn — DISMISSED** [review]

**Review by yqrashawn — APPROVED** [review]

**Pull request merged**